### PR TITLE
refactor(frontend): remove confirmation button from welcome loader

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

We remove the confirmation button from the `Loader` component, and we set it to automatically proceed after a small delay, for visual feedback.

# Tests

In local replica worked as expected:

https://github.com/user-attachments/assets/ef6ee3ce-b31d-4e5a-98f8-57b0ec8a8078

### CODE REVIEW

File: src/frontend/src/lib/components/core/Loader.svelte [hunk 1]
```diff
@@ -38,7 +38,7 @@
 			return;
 		}
 
-		if (confirm) {
+		if (isNullish(oisy_introduction)) {
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/core/Loader.svelte:41]: Why not just removing this step?

File: src/frontend/src/lib/components/core/Loader.svelte [hunk 2]
```diff
@@ -98,6 +94,9 @@
 		}
 
 		await progressAndLoad();
+
+		// Automatically confirm introduction after a delay for visual feedback.
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/core/Loader.svelte:98]: See previous question - i.e. why adding this?

**Review by peterpeterparker — COMMENTED** [review]
In addition, I think the function `confirmIntroduction` and the variable `oisy_introduction` can be removed.

File: src/frontend/src/lib/components/core/Loader.svelte (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/core/Loader.svelte:41]: done

**Comment by AntonioVentilii-DFINITY** [comment]
I thought we wanted to keep this information in localStorage for other purposes. But we can always re-put it there.

Removed for now

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks

**Pull request merged**